neuromation/api/config_factory.py

```diff
@@ -141,6 +146,25 @@ async def login_with_token(
         )
         self._save(config)
 
+    async def login_with_passed_config(
+        self,
+        config_data: str,
```

Comment:
Don't require `config_data` but get it from `os.environ` if not passed.

Reply:
Done